source/operations_and_monitoring.rst

```diff
@@ -260,137 +274,30 @@ Ansible control host
 Control Plane Monitoring
 ========================
 
-Monasca has been configured to collect logs and metrics across the control
-plane. It provides a single point where control plane monitoring and telemetry
-data can be analysed and correlated.
+The control plane has been configured to collect logs centrally using the EFK
+stack (Elasticsearch, Fluentd and Kibana).
 
-Metrics are collected per server via the `Monasca Agent
-<https://opendev.org/openstack/monasca-agent>`__. The Monasca Agent is deployed
-and configured by Kolla Ansible.
+Monitoring of the control plane is performed by Prometheus. Metrics are
```

Comment:
Generally I'd consider monitoring to be a broader term including logs and telemetry.  Prometheus time-series data is telemetry.  Perhaps consider starting with "Telemetry monitoring..."?

Reply:
That's a good point, I'll change.